README.md

```diff
@@ -24,20 +24,20 @@ The access service exists to help answer the following questions for each person
   - Who has access to my application?
   - How do I setup access for a new application?
   - How do I create a new access group for my application?
-  - How do I give access to a role to one of my application's groups?
+  - How do I give one of my application's groups access to a role?
 
 ## Development Setup
 
-Access is a React+Typescript single-page application (SPA) with a Flask API that connects to the Okta API
+Access is a React and Typescript single-page application (SPA) with a Flask API that connects to the Okta API.
 
 You'll need an Okta API Token from an Okta user with the `Group Admin` and `Application Admin`
 Okta administrator roles granted as well as all Group permissions (ie. `Manage groups` checkbox checked)
-in a custom Admin role. If you want to manage Groups which grant Okta Admin permissions, then the Okta API
+in a custom Admin role. If you want to manage Groups which grant `Okta Admin` permissions, then the Okta API
```

Comment:
Okta Admin shouldn't be highlighted as it isn't referencing an Okta administrator role

```suggestion
in a custom Admin role. If you want to manage Groups which grant Okta administrator permissions, then the Okta API
```